Services/CO.CDP.Organisation.WebApi/Program.cs

```diff
@@ -82,8 +83,9 @@
 builder.Services.AddOrganisationProblemDetails();
 
 builder.Services.AddJwtBearerAndApiKeyAuthentication(builder.Configuration, builder.Environment);
-//builder.Services.AddAuthorization();
-builder.Services.AddOrganisationAuthorization();
+builder.Services.AddAuthorization();
```

Comment:
This seems wrong as you disabled our authorisation mechanism and replaced it with the built in configuration

Reply:
Done